syntax = "proto3";

service CheckService {
  rpc CreateCheck (CreateCheckRequest) returns (CreateCheckResponse);
}

message CreateCheckRequest {
  int32 order_id = 1;
  repeated Item items = 2;
}

message Item {
  string name = 1;
  float price = 2;
  int32 quantity = 3;
}

message CreateCheckResponse {
  string file_path = 1;
}